RecoEcal/EgammaCoreTools/src/EcalClustersGraph.cc

```diff
@@ -0,0 +1,438 @@
+#include "RecoEcal/EgammaCoreTools/interface/EcalClustersGraph.h"
+#include <cmath>
+
+using namespace std;
+using namespace reco;
+using namespace reco::DeepSCInputs;
+
+typedef std::shared_ptr<CalibratedPFCluster> CalibratedClusterPtr;
+typedef std::vector<CalibratedClusterPtr> CalibratedClusterPtrVector;
```

Comment:
Do all the `reco::PFCluster`s come from a single collection, or from different ones ?

Reply:
They come from a single collection `particleFlowClusterECAL`, and they are split internally in EB and EE clusters.